pkg/util/node/nodecache_utils.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2020 Authors of Arktos.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package node
+
+import (
+	"fmt"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	coreinformers "k8s.io/client-go/informers/core/v1"
+	corelisters "k8s.io/client-go/listers/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/klog"
+	"sync"
+	"time"
+)
+
+func GetNodeFromNodelisters(nodeListers map[string]corelisters.NodeLister, nodeName string) (*v1.Node, string, error) {
+	for rpId, nodeLister := range nodeListers {
```

Comment:
> this loop seems discontinue after the first iteration.

I think when node is not find in the lister, it will return NotFoundError (in TODO), this case, we will continue to search next node lister. Otherwise, the node is found and hence return

Reply:
Update to check NotFoundError.